src/pages/evm/staking/UnstakingTab.vue

```diff
@@ -0,0 +1,235 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { ethers } from 'ethers';
+
+import {
+    CURRENT_CONTEXT,
+    getAntelope,
+    useAccountStore,
+    useBalancesStore,
+    useChainStore,
+    useRexStore,
+    useUserStore,
+} from 'src/antelope';
+
+import EVMSidebarPage from 'src/layouts/EVMSidebarPage.vue';
+import EVMChainSettings from 'src/antelope/chains/EVMChainSettings';
+import ConversionRateBadge from 'src/components/ConversionRateBadge.vue';
+import CurrencyInput from 'src/components/evm/inputs/CurrencyInput.vue';
+import { formatWei } from 'src/antelope/stores/utils';
+
+const label = 'current';
+
+const { t: $t } = useI18n();
+const uiDecimals = 2;
+const ant = getAntelope();
+const chainStore = useChainStore();
+const chainSettings = chainStore.currentChain.settings as EVMChainSettings;
+const userStore = useUserStore();
+const balanceStore = useBalancesStore();
+const accountStore = useAccountStore();
+
+const systemToken = chainSettings.getSystemToken();
+const systemTokenSymbol = systemToken.symbol;
+const systemTokenDecimals = systemToken.decimals;
+const stakedToken = chainSettings.getStakedSystemToken();
+const stakedTokenSymbol = stakedToken.symbol;
+const stakedTokenDecimals = stakedToken.decimals;
+
+
+// data
+const oneEth = ethers.BigNumber.from('1'.concat('0'.repeat(systemTokenDecimals)));
+const inputModelValue = ref(ethers.constants.Zero);
+const unstakedRatio = computed(() => chainStore.getUnstakedRatio(label));
+const outputModelValue = computed(() => {
+    if (unstakedRatio.value.isZero()) {
+        return ethers.constants.Zero;
+    }
+    const output = inputModelValue.value.mul(unstakedRatio.value).div(oneEth);
+    return output;
+});
```

Comment:
these two should be moved under line 52 (they are computed)

Reply:
Yes, I saw your PR time ago during exactly that but somehow got "lost in the battle".
So, yeah. Moved the comment up